feather/server/src/initial_handler.rs

```diff
@@ -27,6 +27,7 @@ use uuid::Uuid;
 
 use self::proxy::ProxyData;
 
+const VERSION_STRING: &str = "1.16.5";
```

Comment:
Is this not available as a constant on the protocol crate, if not it should be added there?